resources/db/migration/V2__add_criticality_level.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE ze_data.scope
+ ADD COLUMN s_criticality_level SMALLINT DEFAULT 2 CHECK (s_criticality_level >= 1 AND
```

Comment:
Not sure if this default value will be applied, because when executing `(sql/cmd-create-or-update-scope! {...})` the map key `:criticality_level` is always present, but its value might be `nil. I think it would be better to `merge` the defaults into the parameter map before passing them to the `create-or-update-scope!` command.